### PR TITLE
[TECH-417]SignUp page

### PR DESCRIPTION
**Pull request by PoornaPanda9785**

### CODE REVIEW

File: src/pages/sign-up/SignUp.tsx [hunk 1]
```diff
@@ -1,5 +1,195 @@
...
+    lname: '',
+    phoneNumber: '',
+    password: '',
+  }
+  return (
+    <div className="max-w-lg p-10 m-4 mx-auto">
+      <Formik
+        initialValues={initialValues}
+        onSubmit={(values, actions) => {
+          console.log(values)
```
Comment (JeanMariaJose) [inline src/pages/sign-up/SignUp.tsx:32]: when you log pass word , it should be encrypted

File: src/pages/sign-up/SignUp.tsx [hunk 2]
```diff
@@ -1,5 +1,195 @@
...
+        {({ errors, touched }) => (
+          <Form className="bg-white rounded px-2 pt-2 pb-2 mb-4 max-w-xl">
+            <div className="mb-6">
+              <h1 className="bg text-3xl font-semibold content-start">
+                Create an account
+              </h1>
+            </div>
+            <div className="mb-6">
+              <label
+                className="block text-black text-sm font-semibold mb-2"
```
Comment (abinthomas258) [inline src/pages/sign-up/SignUp.tsx:69]: 'block text-black text-sm font-semibold mb-2' * 5 times in the same file can be given as a combined CSS class using @apply annotation

File: package.json [hunk 3]
```diff
@@ -17,11 +17,15 @@
   "dependencies": {
     "axios": "^0.27.2",
     "date-fns": "^2.29.1",
+    "formik": "^2.2.9",
```
Comment (abinthomas258) [inline package.json:20]: Just analyze and compare if we can use React Hook Form over Formik. 

It appears that React Hook Form has frequent updates over Formik which have last published one year ago.

found some useful blogs on the internet

React Hook Form isolates input components from the others and prevents re-render the form for a single input. It avoids this unnecessary re-rendering. So it is a great advantage in terms of performance compared to Formik which updates every change in every input field

**Review by abinthomas258 — CHANGES_REQUESTED** [review]
- Compare React Hook Form vs Formik
- use @apply for reusing CSS class

**Comment by kalarikkalbabu** [comment]
The tests have failed. Please check the logs and fix the issues.

**Pull request merged**